docs/techincal-guide.md

```diff
@@ -41,12 +41,12 @@ bytes32 public constant CANCEL_EXECUTION_ROLE = keccak256("CANCEL_EXECUTION_ROLE
 
 These roles can be set to another Aragon app or an individual address.
 
-We recommend setting the following roles to the following apps or individuals:
-- `SET_DELAY_ROLE` => TBD
-- `DELAY_EXECUTION_ROLE` => TBD
-- `PAUSE_EXECUTION_ROLE` => TBD
-- `RESUME_EXECUTION_ROLE` => TBD
-- `CANCEL_EXECUTION_ROLE` => TBD
+We recommend setting the following roles to the following apps or externally owned accounts:
+- `SET_DELAY_ROLE` => Voting
+- `DELAY_EXECUTION_ROLE` => Voting
+- `PAUSE_EXECUTION_ROLE` => TokenManager || EOA
+- `RESUME_EXECUTION_ROLE` => TokenManager || EOA
+- `CANCEL_EXECUTION_ROLE` => TokenManager || EOA
```

Comment:
I think you didn;t see the edited comment but we decided to not recommend roles granted to any specific entity, since it can be pretty flexible to set depending on the context.